tree/ntuple/v7/inc/ROOT/RNTuple.hxx

```diff
@@ -392,6 +392,8 @@ public:
    /// Ensure that the data from the so far seen Fill calls has been written to storage
    void CommitCluster();
 
+   std::unique_ptr<REntry> CreateEntry() { return fModel->CreateEntry(); }
```

Comment:
This augment the interface and probably deserve its own commit, doesn't it?

Reply:
Done